assets/state-container-toolkit/0500_daemonset.yaml

```diff
@@ -41,7 +41,7 @@ spec:
           seLinuxOptions:
             level: "s0"
         volumeMounts:
-          - name: driver-install-path
+          - name: driver-root
```

Comment:
No, I had changed this to `driver-root` previously to align with naming this field `driverRoot` in the ClusterPolicy API. But since we renamed it to `driverInstallDir`, I am open to renaming it.

Reply:
To reduce churn, I have kept the `driver-root` name. If preferred, I can rename this volume / volumeMount to `driver-install-dir`  in a follow-up.

Reply:
Yes, let's make it match the field names in the `HostPathsSpec` struct.

Reply:
I have gone ahead and updated the name of this volume to `driver-install-dir` throughout to align with the name of the field in ClusterPolicy.